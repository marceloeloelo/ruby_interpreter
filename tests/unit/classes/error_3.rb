class Auto
	def hola
	end
end

Auto.hola() #Falta hacer new
