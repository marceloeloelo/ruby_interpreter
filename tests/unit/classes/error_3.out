NoMethodError: undefined method `hola' for Auto:Class